e2e-tests/cypress/e2e/journeys/insurance/your-business/company-details/company-details-submit.spec.js

```diff
@@ -0,0 +1,55 @@
+import { companyDetails } from '../../../../pages/your-business';
+import { continueButton } from '../../../../pages/shared';
+import { ERROR_MESSAGES } from '../../../../../../content-strings';
+import partials from '../../../../partials';
+import { ROUTES, FIELD_IDS } from '../../../../../../constants';
+
+const {
+  EXPORTER_BUSINESS: {
+    YOUR_COMPANY: {
+      TRADING_NAME,
+    },
+  },
+} = FIELD_IDS.INSURANCE;
+
+const COMPANY_DETAILS_ERRORS = ERROR_MESSAGES.INSURANCE.EXPORTER_BUSINESS;
+
+describe('Your business - company house search - As an Exporter I want to enter details about my business in \'your business\' section', () => {
```

Comment:
Should be able to do this, save extra formatting

```
"Your business - company house search - As an Exporter I want to enter details about my business in 'your business' section"
```

Reply:
done